pkg/sypgp/sypgp.go

```diff
@@ -1014,38 +1018,35 @@ func (keyring *Handle) importPrivateKey(entity *openpgp.Entity) error {
 		return &KeyExistsError{fingerprint: entity.PrivateKey.Fingerprint}
 	}
 
-	// Check if the key is encrypted, if it is, decrypt it
-	if entity.PrivateKey == nil {
-		return fmt.Errorf("corrupted key, unable to recover data")
-	}
-
-	// Make a clone of the entity
-	newEntity := &openpgp.Entity{
-		PrimaryKey:  entity.PrimaryKey,
-		PrivateKey:  entity.PrivateKey,
-		Identities:  entity.Identities,
-		Revocations: entity.Revocations,
-		Subkeys:     entity.Subkeys,
-	}
+	newEntity := *entity
```

Comment:
Why make a `newEntity`? can you just use `entity`?

Reply:
So for some reasons here was used a copy of entity. I decided to leave it as is. I'll investigate deeper if we can use entity directly.